### DISCUSSION TITLE
Navigation / Masthead not showing?

### DISCUSSION THREAD

**Discussion by JamesTsetsekas** [mmistakes/minimal-mistakes#3577] [Q&A] [answered]
Hey I'm trying to setup a portfolio blog, and I'm having issues I'm not able to get my navigation showing can you help me understand why it does not show? I added links to navigation.yml but they are not showing on the page.

![image](https://user-images.githubusercontent.com/8147662/163516633-28241cbe-3a0b-47ea-a512-f23ad7be545b.png)
![image](https://user-images.githubusercontent.com/8147662/163516692-20ab2007-f0ac-40e1-ac41-04f8e6eb28d9.png)

**Comment by iBug** [comment]
I don't think the `home` layout supports sidebar by default. Compare [`home.html`](https://github.com/mmistakes/minimal-mistakes/blob/2632ff650a6efb0d856a37d675be5f1b63692181/_layouts/home.html) and [`single.html`](https://github.com/mmistakes/minimal-mistakes/blob/2632ff650a6efb0d856a37d675be5f1b63692181/_layouts/single.html#L18).

**Reply by JamesTsetsekas** [reply]
I changed my index.html to single but still no nav?
![image](https://user-images.githubusercontent.com/8147662/163580436-dd57d01a-d0bc-4154-b7fa-d68eac6818e7.png)

**Reply by JamesTsetsekas** [reply]
Im still new Jekyll and Minimal Mistakes but i already like it more than wordpress for my personal website / portfolio but I'm struggling to understand how to enable a nav or menu button at the top of the page. Any pointers would be greatly appreciated.

**Comment by JamesTsetsekas** [comment, marked as answer]
I figured this out just add a _data\navigation.yml
